pkg/deployment/network.go

```diff
@@ -31,6 +31,36 @@ import (
 func ConfigureNetwork(ctx context.Context, helper *helper.Helper, obj client.Object, sshKeySecret string, inventoryConfigMap string, aeeSpec dataplanev1beta1.AnsibleEESpec) error {
```

Comment:
These tasks are not network related - should we rename the file and function or split it out to to a separate (bootstrap.go)? file?

Reply:
Yeah, we should/would do that. However, @slagle told me that there would be major refactoring as part of composable services work.  Therefore, I just added the pre-network[1] edpm tasks here, so that we can have those for the time being.

[1] https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/playbooks/deploy_edpm_pre_network.yml

Reply:
OK, makes sense. I will mark this resolved.